user-guide/Troubleshooting/Procedures/Investigating_Web_Issues.md

```diff
@@ -146,6 +146,7 @@ Visual Overview in web apps has limited functionality:
 - Visual Overview pages are rendered as images containing clickable regions. As a result, some features (e.g. embedding) and some session variable controls are not fully supported.
 - To render a Visual Overview page, [SLHelper](xref:Troubleshooting_SLHelper_exe) loads a virtual instance of DataMiner Cube in its memory. Initial loading of a page can take a long time because SLHelper needs to start Cube, connect to a DMA, and then generate an image.
 - SLHelper may use a significant amount of memory to display Visual Overview pages, especially when multiple users are connected. When Visual Overview pages are not viewed by any user, the virtual DataMiner Cube instance is terminated after a timeout of 5 minutes and the memory is released.
+- The DataMiner Agent you are connected to handles all requests and updates for mobile visual overview, which can lead to performance limitations when handling multiple users or requests. From DataMiner 10.5.2/10.6.0 onwards<!--RN 41434-->, you can implement [load balancing](#load-balancing) to distribute requests and updates across multiple Agents in the DMS, alleviating these performance limitations.
```

Comment:
the problem we are mostly solving is when multiple users with different security context log in. Not sure if we want to make it that specific, but I'd leave out request and keep it at users probably.